app/services/theme.js

```diff
@@ -70,29 +70,36 @@ export default Ember.Service.extend({
 
         return pathPrefix;
     }),
-
-    // The URL for the branded stylesheet
-    stylesheet: Ember.computed('id', function() {
+    assetsPath: Ember.computed('id', function() {
         const id = this.get('id');
 
         if (!id)
             return;
-
-        const prefix = this.get('isDomain') ? '' : '/preprints';
-        const suffix = config.ASSET_SUFFIX ? `-${config.ASSET_SUFFIX}` : '';
-        return `${prefix}/assets/css/${id}${suffix}.css`;
+        return `${config.providerAssetsURL}${id}/`;
```

Comment:
Aaaactually… there should be a javascript library for building URL for parts. We are probably using it somewhere, maybe in Ember OSF or possibly somewhere in preprints. We need to use that for these types of things.